### PR TITLE
[17.0][MIG] contract: Migration to 17.0

### PR DESCRIPTION
**Pull request by peluko00**

### CODE REVIEW

File: contract/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2016-2017 LasLabs Inc.
+# Copyright 2018-2019 ACSONE SA/NV
+# Copyright 2020-2021 Tecnativa - Pedro M. Baeza
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Recurring - Contracts Management",
+    "version": "16.0.1.1.2",
```
Comment (angelmoya) [inline contract/__manifest__.py]: Change version, please

**Comment by pedrobaeza** [comment]
Please include #1026 and put the CI green if you want this to be reviewed.

**Comment by wpichler** [comment]
@peluko00 Could it be that you forgot to commit something ? The current state is not installable...

**Comment by rousseldenis** [comment]
@peluko00 Could you follow https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#technical-method-to-migrate-a-module-from-160-to-170-branch ? Thanks

**Comment by victoralmau** [comment]
Please, include https://github.com/OCA/contract/pull/1042

**Comment by jaydeep32** [comment]
I have recently made many changes from this branch migrated module. I added those #1026  and #1042 in my code with migrated test cases.
I want to push this PR but need help getting a 403 error.
Could you allow me or guide me on how I can upload my change to this PR, or should I make one?

**Comment by pedrobaeza** [comment]
@jaydeep32 you need to open a new pull request from your fork. As @peluko00 seems to not answer, we will supersede this one with yours if it's correct. Please also include #1033 (properly maintaining attribution for each commit).

**Pull request closed without merging**